index.js

```diff
@@ -0,0 +1,49 @@
+// const { default: axios } = require("axios");
+
+
+const select = document.querySelector("select");
+const form = document.querySelector("form");
+const reviewInput = document.querySelector("#review-input");
+const submittedReviewsUl = document.querySelector("#submitted-reviews-ul");
+const movieTitle = document.querySelector("#title-h3");
+const releaseYear = document.querySelector("#release-year-p");
+const movieDescription = document.querySelector("#description-p");
+
+const createMovieSelector = async () => {
+  try {
+    const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+
+    res.data.forEach((movie) => {
+      const option = document.createElement("option");
+      option.innerText = movie.title;
+      select.appendChild(option);
+    });
+  } catch (err) {
+    console.log(err);
+  }
+  select.addEventListener("change", async (e) => {
+    try {
+    //   e.preventDefault();
+      const selectedMovie = e.currentTarget.value;
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+      res.data.forEach((movie) => {
+        if (movie.title === selectedMovie) {
+          movieTitle.innerHTML = movie.title;
+          releaseYear.innerHTML = movie.release_date;
+          movieDescription.innerHTML = movie.description;
+        }
```

Comment:
With the value property added to the option object we can make a call to a single film. The api call here should be to a [single film using its id](https://ghibliapi.herokuapp.com/#tag/Films%2Fpaths%2F~1films~1%7Bid%7D%2Fget).
 To get the id of the film that was chosen by the select dropdown we can utilize the `event` object ==> `e.target.value` or from the select variable `select.value`. The `value` property was used before to store the id of that movie. Once we get the data we just reassign the `innerText` for the title, release date, and description.